lib/generators/module/module_generator.rb

```diff
@@ -50,16 +50,18 @@ def create_additional_files
   # :nocov:
   def update_and_install
     # spec helper add group
-    insert_into_file 'spec/spec_helper.rb', "\tadd_group '#{file_name.camelize}',
-      'modules/#{file_name}/'\n", after: "# Modules\n"
+    insert_into_file 'spec/spec_helper.rb',
+                     "\tadd_group '#{file_name.camelize}', 'modules/#{file_name}/'\n", after: "# Modules\n"
```

Comment:
Typically I make the longer lines the previous lines rather than the last line. Thoughts?

Reply:
I just thought this read a little better